docs/getting-started/04-expose-microservice.md

```diff
@@ -0,0 +1,188 @@
+
+## Reference
+
+This guide demonstrates how [API Gateway](/components/api-gateway) works in Kyma. It exposes your Service under secured or unsecured endpoints for external Services to communicate with it.
+
+## Steps
+
+### Expose the Service
+
+Create an APIRule CR which exposes the Kubernetes Service of the microservice under na unsecured endpoint (**handler** set to `noop`) and accepts the `GET` and `POST` methods.
+
+>**TIP:** **noop** stands for "no operation" and means access without any token. If you want to secure your Service, read the [tutorial](/components/api-gateway/#tutorials-expose-and-secure-a-service) to learn how to do that.
+
+Follow these steps:
+
+<div tabs name="steps" group="expose-microservice">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Open the terminal window and apply the [APIRule CR](/components/api-gateway#custom-resource-api-rule):
+
+  ```yaml
+  cat <<EOF | kubectl apply -f -
+  apiVersion: gateway.kyma-project.io/v1alpha1
+  kind: APIRule
+  metadata:
+    name: orders-service
+    namespace: orders-service
+    labels:
+      app: orders-service
+      example: orders-service
+  spec:
+    service:
+      host: orders-service
+      name: orders-service
+      port: 80
+    gateway: kyma-gateway.kyma-system.svc.cluster.local
+    rules:
+      - path: /.*
+        methods: ["GET","POST"]
+        accessStrategies:
+          - handler: noop
+        mutators: []
+  EOF
+  ```
+2. Check if the API Rule was created and has the `OK` status:
+
+   ```bash
+   kubectl get apirules orders-service -n orders-service -o=jsonpath='{.status.APIRuleStatus.code}'
+   ```
+
+</details>
+<details>
+<summary label="ui">
+UI
+</summary>
+
+>**TIP:** You can expose a Service or a Function with an API Rule from different views in the Console UI. This tutorial shows how to do that from the generic **API Rules** view.
+
+1. Select the `orders-service` Namespace from the drop-down list in the top navigation panel.
+
+2. Go to **Configuration** > **API Rules** in the left navigation panel, and select **Create API Rule**.
+
+3. In the **General settings** section:
+
+    - Enter `orders-service` as the API Rule's **Name**.
+
+    >**NOTE:** The APIRule CR can have a different name than the Service, but it is recommended that all related resources share a common name.
+
+    - Enter `orders-service` as **Hostname** to indicate the host on which you want to expose your Service.
+
+    - Select `orders-service (port: 80)` from the drop-down list in the **Service** column to indicate the Service name for which you want to create the API Rule.
+
+4. In the **Access strategies** section, leave only the `GET` and `POST` methods marked and the `noop` handler selected. This way you will be able to send the orders to the Service and retrieve orders from it without any token.
+
+5. Select **Create** to confirm the changes.
+
+    The message will appear on the screen confirming the changes were saved.
```

Comment:
Ok, I changed it everywhere else treating docs as separately, since this message appears in other guides as well. Or do you rather thing in other it should be already `the`?

Reply:
Thanks!
I would treat them separately, since one might not read them in one go, or might already know some stuff and just refer to a portion of these guides.
They're really well-written! I believe they'll be used much more than just for getting started!